setupinfo.py

```diff
@@ -445,6 +434,17 @@ def get_library_versions():
     sys.exit(1)
 
 
+def check_build_dependencies():
+    xml2_version, xslt_version = get_library_versions()
+    if (not check_min_version(xml2_version, '2.7.0', 'libxml2') or
+            not check_min_version(xslt_version, '1.1.23', 'libxslt')):
+        sys.exit(1)
```

Comment:
I know, this is part of a build script, but I'd rather not see this _function_ kill the runtime. A return value is always better for a function like this.

Reply:
Doesn't that make it possible that the build might fail if a user's version of libxml2 and/or libxslt is too old? That said, it's probably unlikely, as those versions have been available for several years now.

Would you be happy with returning True/False, then from the caller, raising an exception? I'll push that version.

Reply:
> from the caller, raising an exception

Yes, that's what I meant. There's probably a place higher up where raising such an exception would be expected.

Reply:
Okay. I've updated the MR.